File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int i = 0;
        int j = 1;
        int count = 1;

        while(j<nums.length){
            if(nums[i]!=nums[j]){
                nums[i+1]=nums[j];
                count++;
                i++;
            }
            j++;
        }
        return count;
        
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        Lp=0
        Rp=1

        for Rp in range(len(nums)):
            if(nums[Lp]!=nums[Rp]):
                Lp+=1
                nums[Lp]=nums[Rp]
        
        return Lp+1
        